qbft/spectest/tests/msg_processing_spectest.go

```diff
@@ -21,6 +20,7 @@ type MsgProcessingSpecTest struct {
 	Name               string
 	Pre                *qbft.Instance
 	PostRoot           string
+	PostState          types.Root `json:"-"` // Field is ignored by encoding/json
```

Comment:
@alonmuroch, a little question to you (not for this PR):
Is it really bad to include `PostState` in the json?

Even though `PostRoot` is shorter I see an advantage that the json will describe the full post state in a human readable format

Reply:
@olegshmuelov @GalRogozinski this means that json tests will not be able to do state comparison